client/asset/eth/eth.go

```diff
@@ -3588,7 +3606,11 @@ func (w *TokenWallet) sendToAddr(addr common.Address, amt uint64, maxFeeRate *bi
 			return err
 		}
 		tx, err = c.transfer(txOpts, addr, w.evmify(amt))
```

Comment:
How interesting that we're using `w.node.txOpts` *inside* `withTokenContractor` in this method.  Is there a reason?

Reply:
I cannot think of a reason, moving it above.